searx/static/themes/simple/src/less/style.less

```diff
@@ -330,38 +334,38 @@ article[data-vim-selected].category-social {
   font-style: italic;
 }
 
+.result-images:last-child {
+  flex-grow: 10;
+}
+
 .result-images {
-  display: inline-block;
-  margin: 0;
-  padding: 0;
-  position: relative;
-  max-height: 200px;
+  flex-grow: 1;
+  padding: 0.5rem 0.5rem 2rem 0.5rem;
+  margin: 0.25rem;
   border: none !important;
+  height: @results-image-row-height;
 
   img {
-    float: inherit;
-    margin: 0.125rem;
+    margin: 0;
     padding: 0;
     border: none;
-    max-height: 200px;
+    max-height: 100%;
+    min-width: 100%;
+    object-fit: cover;
+    vertical-align: bottom;
     background: var(--color-result-image-background);
+    width: 100%;
   }
 
   span.title {
-    display: none;
-    color: var(--color-result-image-span-font);
-  }
-
-  &:hover span.title {
     display: block;
-    position: absolute;
-    bottom: 0;
-    right: 0;
-    padding: 4px;
-    margin: 0 0 4px 4px;
-    // color: @color-result-image-span-font;
-    background-color: var(--color-result-image-span-background-hover);
-    font-size: 0.7em;
+    font-size: 0.9rem;
+    color: var(--color-result-image-span-font);
+    padding: 0.5rem 0 0 0;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    max-width: 12rem;
```

Comment:
It may not work with narrowed image.

A possible solution:
```less
.result-images {
  & > a {
    position: relative;

    & > span.title {
	  position: absolute; // combined with relative above, width: 100% doesn't overflow the a element.
   	  width: 100%;
    }
  }
}

```

Reply:
Thanks; I updated the commit with these changes included :+1: